tests/e2e/specs/global/tools.spec.dev.js

```diff
@@ -0,0 +1,21 @@
+function setup() {
+  cy.wipeDatabase();
+  cy.visit('/', {
+    onBeforeLoad(win) {
+      cy.stub(win.console, 'log').as('consoleLog')
+      cy.stub(win.console, 'error').as('consoleError')
+    },
```

Comment:
We stub the console now so we can listen for the console log from the devtools connection instead of watching for the element. 😎